src/App.tsx

```diff
@@ -89,10 +89,22 @@ const Settings = React.lazy(() =>
 import { withGlobalContext } from 'hooks/useGlobalState'
 import { rootReducer, INITIAL_STATE } from 'reducers-actions'
 import PrivateRoute from './PrivateRoute'
+import { assertNonNull } from 'utils'
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 const Router: typeof BrowserRouter & typeof HashRouter = (window as any).IS_IPFS ? HashRouter : BrowserRouter
 
+function getInitialUrl(): string {
+  // Validate tha
```

Comment:
?
```suggestion
  // Validate that
```

Reply:
this comment is supposed to be deleted i guess